### ISSUE TITLE
Database Connection Not Close After Command Ended

### ISSUE THREAD

**Issue by muhdfaiz** [Pop-Code/nestjs-console#275]
Hi, I have an issue with the database connection not close after running any command.

Every time I run the command, it will create a new database connection until I got an error `Too Many Connection`.

It only happens when I running the command. If not running command, the database connection will be terminated when I shut down the app.

I checked the BootstrapConsole code and I can't see the lifecycle hook init. Is that the reason?

How I use `prisma` is I follow the basic step here. https://docs.nestjs.com/recipes/prisma

Here I attached the gif image for your reference.

![Jan-21-2021-09-27-44](https://user-images.githubusercontent.com/18674455/105268794-c8c97000-5bcd-11eb-87ff-54aeb71b80ff.gif)

**Nest.js version:** 7.5.2
**ORM:** [Prisma](https://www.prisma.io/)
**Database:** MySQL

**prisma.service.ts**

```
import { Injectable, OnModuleInit, OnModuleDestroy } from '@nestjs/common';
import { PrismaClient } from '@prisma/client';

@Injectable()
export class PrismaService
  extends PrismaClient
  implements OnModuleInit, OnModuleDestroy {
  constructor() {
    super({
      log: [{ emit: 'event', level: 'query' }],
    });
  }

  async onModuleInit() {
    await this.$connect();
  }

  async onModuleDestroy() {
    await this.$disconnect();
  }
}
```

**app.module.ts**

```
@Module({
  imports: [ConsoleModule, TestModule],
  providers: [AppGateway],
})
export class AppModule {}
```

**console.ts**

```
import { BootstrapConsole } from 'nestjs-console';
import { AppModule } from './app.module';

const bootstrap = new BootstrapConsole({
  module: AppModule,
  useDecorators: true,
});

bootstrap.init().then(async (app) => {
  try {
    await app.init();

    await bootstrap.boot();

    process.exit(0);
  } catch (e) {
    process.exit(1);
  }
});
```

**test.service.ts (Command)**

```
import { HttpService } from '@nestjs/common';
import { Command, Console } from 'nestjs-console';
import { PrismaService } from 'src/prisma/prisma.service';

@Console()
export class TestService {
  constructor(
    private httpService: HttpService,
    private prisma: PrismaService,
  ) {}

  @Command({
    command: 'test:get',
    description: 'test command',
  })
  async testCommand() {
    const news = await this.prisma.news.findMany();

    console.log('execute test command.');
  }
}
```

**Comment by muhdfaiz** [comment]
I have found the cause that causing database connection not terminate when the command finish or error. It's because of the process.exit().

I already make a pull request contain a readme update to use `app.close()` instead of `process.exit()`. You can view the pull request here [#276](https://github.com/Pop-Code/nestjs-console/pull/276)

Thank you.

**Comment by Rmannn** [comment]
fixed by #276 thanks
